### ISSUE TITLE
International Registry of Seismograph Stations (IR) compatibility?

### ISSUE THREAD

**Issue by krischer** [iris-edu/mseed3-evaluation#27]
By chance I just stumbled across this:

http://www.isc.ac.uk/registries/download/IR_implementation.pdf

I must admit that I am not super familiar with it so maybe someone else can chime in. Does anyone actually use that?

My question is if we should make this the default stream identifier?

It is essentially `Agency.Deployment.Station.Location.Channel` where `Agency` would always be `FDSN` for SEED compatibility and the `Deployment` code is the `Network` code. So it would be a minor effort to include it but it might make the new MiniSEED more compatible.

**Comment by crotwell** [comment]
Humm, nice when we have 2 standards organizations not knowing what the other is doing! :(

Just using it has the advantage of we are not creating the url, just adopting, which is nice. But on the other hand they say this:

> Location: 0-2 alphanumeric characters (e.g., SEED v2.4 compatible).
> Channel: 3 alphanumeric characters following the SEED v2.4 coding standard.

So we cannot adopt this directly and solve the large-N issue or the new instrument type issue.

They also double dot a null loc code, so you get `FDSN.US.DUG..BHZ`. I find that ugly, but maybe it is acceptable.

**Comment by chad-iris** [comment]
This alternate identifier concept was discussed some 10 years ago now.  The FDSN decided not to adopt it and I think there is some kind of mapping worked out between the two systems.  It may be in use at parametric data centers but I do not know of any use for waveform identification.

> My question is if we should make this the default stream identifier?

IMHO no.  Even if we wanted to discuss it again it should be in a broad FDSN context as if effects everything FDSN, formats, documentation, web pages, etc.

**Comment by chad-iris** [comment]
> It is essentially Agency.Deployment.Station.Location.Channel where Agency would always be FDSN for SEED compatibility and the Deployment code is the Network code. So it would be a minor effort to include it but it might make the new MiniSEED more compatible.

Right, I should finished reading!  That's the mapping that was worked out.  I don't see any value in adding it.  The IDs we've discussed are more or less the same, with the expansions we need.
